### ISSUE TITLE
Everything is invisible and only a shadow fills the square wherever you put what you chose.

### ISSUE THREAD

**Issue by Vaerion** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#912]
**Describe the bug**
Here tell us what you thought was meant to happen and then how TaleSpire vomited on itself

**To Reproduce**
Run the program with these specs;
    Edition: Windows 10 Pro
    Version: 20H2
    OS Build: 19042.928
    Experience: Windows Feature Experience Pack 120.2212.551.0
    
    Processor: Intel(R) Core(TM) i3-2120 CPU @ 3.30GHz   3.30 GHz
    Installed RAM: 8.00 GB (7.90 GB usable)
    System Type: 64-bit operating system, x64-based processor

**Additional context**
There are multiple Platform does not support compute shaders in the error messages.

**Comment by LuPro** [comment]
Can you check issue #779 for the steps to get to your dxdiag page? I think your CPU's integrated graphics might unfortunately just be a bit too old and doesn't support features needed by TS (You need directx 11.0 shader model 5 and I think that integrated GPU is only directx 10.1 shader model 4.1)

**Comment by Vaerion** [comment]
Sure how do I look at that? Apologies, I'm not a programmer.

Sent from Yahoo Mail on Android 
 
  On Sun, May 9, 2021 at 5:19 PM, ***@***.***> wrote:   


Can you check issue #779 for the steps to get to your dxdiag page? I think your CPU's integrated graphics might unfortunately just be a bit too old and doesn't support features needed by TS (You need directx 11.0 shader model 5 and I think that integrated GPU is only directx 10.1 shader model 4.1)

**Comment by Vaerion** [comment]
It says DirectX 12

Sent from Yahoo Mail on Android 
 
  On Sun, May 9, 2021 at 5:19 PM, ***@***.***> wrote:   


Can you check issue #779 for the steps to get to your dxdiag page? I think your CPU's integrated graphics might unfortunately just be a bit too old and doesn't support features needed by TS (You need directx 11.0 shader model 5 and I think that integrated GPU is only directx 10.1 shader model 4.1)

**Comment by LuPro** [comment]
You need to look in the second tab (Should be called "Display" or something like it), the first page always says DirectX 12 because Windows 10 supports that (but your hardware might not)

**Comment by Vaerion** [comment]
Direct3DDI: 11.1?

Sent from Yahoo Mail on Android 
 
  On Sun, May 9, 2021 at 5:51 PM, ***@***.***> wrote:   


You need to look in the second tab (Should be called "Display" or something like it), the first page always says DirectX 12 because Windows 10 supports that (but your hardware might not)

**Comment by Vaerion** [comment]
At the very bottom of the first tab is DXDiag 10.00.19041.0928

Sent from Yahoo Mail on Android 
 
  On Sun, May 9, 2021 at 5:51 PM, ***@***.***> wrote:   


You need to look in the second tab (Should be called "Display" or something like it), the first page always says DirectX 12 because Windows 10 supports that (but your hardware might not)

**Comment by LuPro** [comment]
Yeah, that's a DX 10.x version, also googled around a bit and it seems your integrated graphics (Intel hd 2000) only support directx feature level 10_1 and only with feature level 11_0 there is support for shader model 5 (which is required by the game)
If you don't have a dedicated graphics card that is a bit newer unfortunately your hardware doesn't seem to support TS :|

**Comment by Vaerion** [comment]
So, if I get a graphics card, which is in the plan, I should be good?

Sent from Yahoo Mail on Android 
 
  On Sun, May 9, 2021 at 6:32 PM, ***@***.***> wrote:   


Yeah, that's a DX 10.x version, also googled around a bit and it seems your integrated graphics (Intel hd 2000) only support directx feature level 10_1 and only with feature level 11_0 there is support for shader model 5 (which is required by the game)
If you don't have a dedicated graphics card that is a bit newer unfortunately your hardware doesn't seem to support TS :|

**Comment by LuPro** [comment]
I'm not entirely sure if the CPU performance is high enough for big boards, but I *think* the CPU as a CPU is good enough - it's just that the GPU in the CPU is not up to snuff. Before committing to anything I'd first research a bit more if some other features aren't supported by the CPU which might lose a problem - I'll have to come back to you later on this, it's getting quite late here and I don't think I'll be able to find that info quickly
